src/Graphics/Vty/Image/Internal.hs

```diff
@@ -82,7 +82,7 @@ data Image =
     | HorizJoin
       { partLeft :: Image 
       , partRight :: Image
-      , outputWidth :: Int -- ^ imageWidth partLeft == imageWidth partRight. Always > 1
+      , outputWidth :: Int -- ^ imageWidth partLeft == imageWidth partRight. Always > 0
```

Comment:
I'm not sure that this is correct, if you HorizJoin two EmptyImages isn't outputWidth zero?